### DISCUSSION TITLE
Is it possible to set value for RadioButtonList with hidden inputs?

### DISCUSSION THREAD

**Discussion by AngryTester95** [atata-framework/atata#563] [Q&A] [answered]
Hi @YevgeniyShunevych!

I have a html control:
![image](https://user-images.githubusercontent.com/82048855/138130881-fac083b3-38c2-4dcf-8199-cb748283ce2d.png)

And I'd like to describe it as RadioBattonList<TOwner>, but inputs are hidden.  Is it possible somehow to set value to hidden element from RadioBattonList<TOwner> control?

**Comment by YevgeniyShunevych** [comment, marked as answer]
Hi @AngryTester95,

I don't see quick solution to configure `RadioButtonList` currently to work with hidden radio buttons. What you can do is to use a couple of `RadioButton` controls instead. Kind of:

```cs
[FindByValue("OPTIMIZED", Visibility = Visibility.Any)]
[ClicksUsingActions]
public RadioButton<_> OptimizedOnly { get; private set; }

[FindByValue("ORIGINAL", Visibility = Visibility.Any)]
[ClicksUsingActions]
public RadioButton<_> OriginalsOnly { get; private set; }
```

You can also group them to a separate control class.

**Reply by AngryTester95** [reply]
@YevgeniyShunevych  Yes, I have thought regarding it as well. Thank you!

**Reply by YevgeniyShunevych** [reply]
You are welcome.
